libfabric.spec

```
@@ -1,8 +1,8 @@
 %define suse_libname libfabric1
 
 Name: libfabric
-Version: 1.11.0
-Release: 2%{?dist}
+Version: 1.11.1rc1
```

```suggestion
Version: 1.11.1~rc1
```

The `~` is significant in that it tells RPM that a version with a `~` is < a version without, so that, in this case, 1.11.1 will be seen as an upgrade to 1.11.1~*.

---

done